src/snowflake/snowpark/modin/pandas/snow_partition_iterator.py

```diff
@@ -5,10 +5,9 @@
 from collections.abc import Iterator
 from typing import Any, Callable
 
+import modin.pandas.dataframe as DataFrame
 import pandas
 
-import snowflake.snowpark.modin.pandas.dataframe as DataFrame
```

Comment:
@sfc-gh-joshi i think you will need to move this file down to the extensions folder later